lib/course_cache_manager.rb

```diff
@@ -67,7 +67,7 @@ def update_trained_count
   end
 
   def update_revision_count
-    @course.revision_count = @course.revisions.live.size
+    @course.revision_count = @course.tracked_revisions.live.size
```

Comment:
There's more in this file that needs to be adjusted to ensure that only tracked articles are contributing to the stats.

The main before/after test should look at every one of these that are based on articles edited: `revision_count`, `character_sum_ms`, `view_sum`, `references_count`, `article_count`, `new_article_count`.